packages/shopware-6-client/__tests__/endpoints.spec.ts

```diff
@@ -0,0 +1,242 @@
+import {
+  PRODUCT_ENDPOINT,
+  PRODUCT_DETAILS_ENDPOINT,
+  CATEGORY_ENDPOINT,
+  CATEGORY_DETAILS_ENDPOINT,
+  CUSTOMER_ADDRESS_ENDPOINT,
+  CUSTOMER_ADDRESS_DETAILS_ENDPOINT,
+  CUSTOMER_ADDRESS_SET_DEFAULT_SHIPPING_ENDPOINT,
+  CUSTOMER_ADDRESS_SET_DEFAULT_BILLING_ENDPOINT,
+  CUSTOMER_ENDPOINT,
+  CUSTOMER_DETAILS_ENDPOINT,
+  CUSTOMER_LOGIN_ENDPOINT,
+  CUSTOMER_LOGOUT_ENDPOINT,
+  CUSTOMER_ORDER_ENDPOINT,
+  CHECKOUT_CART_ENDPOINT,
+  CHECKOUT_CART_PRODUCT_ENDPOINT,
+  CHECKOUT_CART_LINE_ITEM_ENDPOINT,
+  CHECKOUT_ORDER_ENDPOINT,
+  CHECKOUT_GUEST_ORDER_ENDPOINT,
+  CHECKOUT_GUEST_ORDER_DETAILS_ENDPOINT,
+  CHECKOUT_ORDER_PAY_ENDPOINT,
+  CONTEXT_CURRENCY_ENDPOINT,
+  CONTEXT_LANGUAGE_ENDPOINT,
+  CONTEXT_COUNTRY_ENDPOINT,
+  CONTEXT_PAYMENT_METHOD_ENDPOINT,
+  CONTEXT_SHIPPING_METHOD_ENDPOINT,
+  CONTEXT_SALUTATION_ENDPOINT,
+  NEWSLETTER_SUBSCRIBE_ENDPOINT,
+  NEWSLETTER_UNSUBSCRIBE_ENDPOINT
+} from "../src/endpoints";
+
+const sampleProductId = "eea0f69ec02d44f7a4224272b3d99478";
+const sampleCategoryId = "03dfd5badd3d43bd8a345ef660761e09";
+const sampleAddressId = "324af469318f46b68e0fe69d77ef15fb";
+const sampleCustomerId = "8b67c1fbb718487db750651430023298";
+const sampleLineItemId = "042df8812942487bb52c9fc1e5b26e20";
+const sampleOrderId = "27356105cf9b4484b96143881c37bbcb";
+
+describe("endpoints", () => {
+  describe("PRODUCT_ENDPOINT", () => {
+    it("should return Shopware product endpoint", async () => {
+      const result = await PRODUCT_ENDPOINT();
+      expect(result).toEqual("/product");
+    });
```

Comment:
Endpoint methods does not return a promise :)
We shouldn't `await` for non-promise method

```ts
    it("should return Shopware product endpoint", () => {
      const result = getProductEndpoint();
      expect(result).toEqual("/product");
    });
```